aws_lambda_powertools/utilities/idempotency/serialization/custom_dict.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any, Callable, Dict
+
+from aws_lambda_powertools.utilities.idempotency.serialization.base import BaseDictSerializer
+
+
+class CustomDictSerializer(BaseDictSerializer):
+    def __init__(self, to_dict: Callable[[Any], Dict], from_dict: Callable[[Dict], Any]):
+        """
+        Parameters
+        ----------
+        to_dict: Callable[[Any], Dict]
+            A function capable of transforming the saved data object representation into a dictionary
+        from_dict: str
```

Comment:
I believe this should be

```suggestion
        from_dict: Callbable[[Dict], Any]
```